apps/src/applab/designElements/CopyElementToScreenButton.jsx

```diff
@@ -91,7 +92,8 @@ class CopyElementToScreenButton extends React.Component {
           className={style.copyElementToScreenButton}
           onClick={this.handleDropdownClick}
         >
-          Copy to screen <i className="fa fa-chevron-down" />
+          {applabMsg.designWorkspace_copyToScreenButton()}
+          <i className="fa fa-chevron-down" />
```

Comment:
@artem-vavilov Forgive me for not knowing this off the top of my head, but can you confirm you didn't lose a trailing space after "Copy to screen" here? It shouldn't be `{`${applabMsg.designWorkspace_copyToScreenButton()} `} `<i className="fa fa-chevron-down" />`, should it?

Reply:
<img width="165" alt="Screenshot 2023-06-21 at 03 18 49" src="https://github.com/code-dot-org/code-dot-org/assets/11708250/53ad5d34-2e25-4175-b658-2a89e1b595ed">
<img width="778" alt="Screenshot 2023-06-21 at 03 18 41" src="https://github.com/code-dot-org/code-dot-org/assets/11708250/66a98f52-bd40-48b4-bb54-4e026d61649a">